elodie/media/media.py

```diff
@@ -207,6 +211,22 @@ def set_date_taken(self, time):
         self.reset_cache()
         return status
 
+    def set_address(self, location):
```

Comment:
I think this should be named differently. We're specifically setting tags with address information, right?

I propose `set_address_tags`.